etl_manager/meta.py

```diff
@@ -756,3 +783,16 @@ def read_database_folder(folderpath):
         tm = read_table_json(table_file_path, database=db)
         db.add_table(tm)
     return db
+
+def get_existing_database_from_glue_catalogue(database_name):
```

Comment:
I would add a docstring here making it clear that it returns a DatabaseMeta Object without the tables (regardless if the tables existed or not)

Reply:
Done